source/Government.cpp

```diff
@@ -115,30 +140,47 @@ void Government::Load(const DataNode &node)
 						grand.PrintTrace("Skipping unrecognized attribute:");
 				}
 		}
-		else if(child.Token(0) == "bribe" && child.Size() >= 2)
-			bribe = child.Value(1);
-		else if(child.Token(0) == "fine" && child.Size() >= 2)
-			fine = child.Value(1);
-		else if(child.Token(0) == "enforces" && child.HasChildren())
+		else if(key == "provoked on scan")
+			provokedOnScan = true;
+		else if(key == "enforces" && child.HasChildren())
 			enforcementZones.emplace_back(child);
-		else if(child.Token(0) == "enforces" && child.Size() == 2 && child.Token(1) == "all")
+		else if(!hasValue)
+			child.PrintTrace("Error: Expected key to have a value:");
+		else if(key == "display name")
+			displayName = child.Token(valueIndex);
```

Comment:
I would expect the `add` keyword to only be valid for entries where `add` actually has a separate meeting from the non-add version. This is especially important for things like `"friendly hail"`, where `add "friendly hail"` might get a meaning in the future to mean that the existing hails stay, and more get added (instead of the hails getting replaced).

Reply:
So, essentially, consider anything that shouldn't be possible with an add as a mistake?

Reply:
The code-change tells me that you understood me perfectly. Thanks!